libs/ui-libraries/src/lib/ui-toolbar/ui-toolbar.component.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../../../theme/theme.scss';
+
+mat-toolbar {
+    display: flex;
```

Comment:
Also, styles in this file will leak out, you need to encapsulate your styles in here now.